README.rst

```diff
@@ -43,6 +43,24 @@ Though conceptually names differ between cloud and infrastructure virtualization
 it was decided to stick to one representation in the codebase and interface to give a singlar API across
 all systems.
 
+Installation
+------------
+
+Wrapanapi can be installed via `pip` as `pip install wrapanapi`
+It is always a good idea to use virtualenv to install pip packages.
+
+For Linux Users, Depending on the distribution you are using, you may need to install following packages
+(or similar for your distrobution of linux):
```

Comment:
I'm confused, is `distrobution` a typo or intended? It kinda looked like a typo at first, but now I like that word :D

Reply:
Lol that's a typo. I keep saying "Distro" in short. hence. I will fix it.